src/test/test_utils.py

```diff
@@ -38,11 +39,17 @@
                           ("wav", ["audio/x-wav"]),
                           ("wma", ["audio/x-ms-wma", "video/x-ms-wma",
                                    "video/x-ms-asf"])])
+
 def testSampleMimeTypes(ext, valid_types):
     guessed = guessMimetype(os.path.join(DATA_D, "sample.%s" % ext))
     if guessed:
         assert guessed in valid_types
 
+def testSpecChars(ext, valid_types):
+    if sys.platform == "win32":
```

Comment:
Ext and valid_types are not being supplied since it not paramertized like testSampleMimeTypes. Thus, the test fails since pytest cannot find fixtrures names ext or valid_types.